lib/membrane/rtcp/parser.ex

```diff
@@ -24,7 +24,13 @@ defmodule Membrane.RTCP.Parser do
 
   @impl true
   def handle_caps(:input, _caps, _ctx, state) do
-    {{:ok, caps: {:output, %RTCP{}}}, state}
```

Comment:
Caps are needed only when we want to send buffers and we don't send any buffers over `:output` pad. We send only events. When I moved sending these caps to `handle_prepared_to_playing`, SSRCRouter crashed as it is not prepared for receiving RTCP caps, only RTP ones. I think that `handle_caps` callback has been never called in this element. Not sure what we want to do with this? IMO, we should fix SSRCRouter but I would move this to a separate PR to not block releasing other packages.

cc @mat-hek

Reply:
I think caps may be useful no matter if we send buffers or not. Maybe one day we should enforce sending them before sending any events as well? I guess fixing SSRCRouter is the proper way for this case 👍